packages/dataviews/src/constants.js

```diff
@@ -28,26 +28,17 @@ export const VIEW_LAYOUTS = [
 		label: __( 'Table' ),
 		component: ViewTable,
 		icon: blockTable,
-		supports: {
```

Comment:
This means there is no longer a need to export anything from dataviews (`VIEW_LAYOUTS` or any specific `getViewSupport` function).